cmd/root/results_test.go

```diff
@@ -438,6 +438,20 @@ func TestMostMissedKeys(t *testing.T) {
 			events: []event{},
 			want:   "",
 		},
+		{
+			name: "no mistakes",
+			events: parseEvents(
```

Comment:
Can you even consider the amount of events that you have parsed in this file? Holy DUMP.